components/BlocksTable/BlocksTable.tsx

```diff
@@ -57,16 +57,26 @@ const BlocksTable: FC<BlocksTableProps> = props => {
   const router = useRouter()
 
   return (
-    <CommonTable
-      {...props}
-      columns={COLUMNS}
-      onRowClick={(block: BlockType) =>
-        router.push({
-          pathname: RoutePaths.BlockInfo,
-          query: { id: block?.sequence.toString() },
-        })
-      }
-    />
+    // TODO: can remove container div once the Phase 3 badge is gone
+    // (https://airtable.com/appIXmGgVqP9QdbCf/tblAWSTOQbTq6vh6X/viwVMBWs7bGdpLiOd/reczg8YuO3jIBQ3pj)
+    <div>
+      <InfoBadge
+        mb="1rem"
+        p=".75rem"
+        w="100%"
+        message="Testnet Phase 2 is complete! Stay tuned for Testnet Phase 3, launching January 18, 2023."
```

Comment:
There's no wrapping here on mobile or small width because this by default has whitespace: nowrap here.

```suggestion
        message="Testnet Phase 2 is complete! Stay tuned for Testnet Phase 3, launching January 18, 2023."
        whiteSpace={'normal'}
```

**Before**
![Screen Shot 2023-01-06 at 5 13 18 PM](https://user-images.githubusercontent.com/458976/211108867-72e6b63c-8339-483c-ae9a-c225a304e035.png)

**After**
![Screen Shot 2023-01-06 at 5 13 25 PM](https://user-images.githubusercontent.com/458976/211108861-2ce6a368-8d74-4f77-9ec7-0178c4663474.png)